index.html

```diff
@@ -0,0 +1,168 @@
+              <input type="text" class="search-form__input-search" />
+            </form>
+          </div>
+        </nav>
+      </section>
+      <section class="header__section">
+        <div class="header__content">
+          <h1 class="display-2 header__title">What will your verse be?</h1>
+          <h3 class="display-5 header__desc"
+            >Everyone has something to share. See how people are using iPad in
+            their unique ways to add to the world’s story.</h3
+          >
+          <h5 class="display-5 header__video-link"
+            >Watch the “Your Verse” film and explore the stories</h5
+          >
+        </div>
+      </section>
+    </header>
+    <section class="featured"
+      ><h6>&nbsp;</h6>
+      <div class="featured__item">
+        <a href="#" class="featured__item__1-link">
+          <img
+            src="./img/promo_iphone5s_2x.jpg"
+            alt=""
+            class="featured__item__ featured__item__1"
+          />
+        </a>
+      </div>
+      <div class="featured__item">
+        <a href="#" class="featured__item__2-link">
+          <img
+            src="./img/promo_iphone5c_2x.jpg"
+            alt=""
+            class="featured__item__ featured__item__2"
+          />
+        </a>
+      </div>
+      <div class="featured__item">
+        <a href="#" class="featured__item__3-link">
+          <img
+            src="./img/promo_30_years_2x.jpg"
+            alt=""
+            class="featured__item__ featured__item__3"
+          />
+        </a>
+      </div>
+      <div class="featured__item">
+        <a href="#" class="featured__item__4-link">
+          <img
+            src="./img/promo_supplier_responsibility_2x.jpg"
+            alt=""
+            class="featured__item__ featured__item__4"
+          />
+        </a>
+      </div>
+    </section>
+    <footer class="footer">
+      <div class="footer__div footer__text">
+        <p class="footer__text-1">
+          Shop the Apple Online Store(1-800-MY-APPLE), visit an Apple Retail
+          Store, or find a reseller.
+        </p>
+        <p class="footer__text-2">
+          Copyright © 2014 Apple Inc. All rights reserved. Terms of Use |
+          Privacy Policy
+        </p>
+      </div>
+
+      <div class="footer__div footer__site-map">
+        <div class="footer__text-3">
+          <div class="footer__text-3-item site">
+            Site Map
+          </div>
+          <div class="footer__text-3-item hot">
+            Hot News
+          </div>
+          <div class="footer__text-3-item rss">
+            RSS Feeds
+          </div>
+          <div class="footer__text-3-item media">
+            Media Info
+          </div>
+          <div class="footer__text-3-item env">
+            Environment
+          </div>
+          <div class="footer__text-3-item job">
+            Job Opportunities
+          </div>
+          <div class="footer__text-3-item contact">
+            Contact Us
+          </div>
```

Comment:
'Site Map', 'Hot News', 'RSS Feeds', 'Mafia Info', 'Environment', 'Job Opportunities', 'Contact Us' - should be links

Reply:
done